02-peer-protocol.md

```diff
@@ -780,6 +817,12 @@ For a timed out or route-failed HTLC:
    * [`2`:`len`]
    * [`len`:`reason`]
 
+The `reason` field is an opaque encrypted blob for the benefit of the
+original HTLC initiator as defined in [BOLT #4](04-onion-routing.md);
+however, there's a special malformed failure variant for the case where
+our peer couldn't parse it: in this case the current node instead take action, encrypting
```

Comment:
s/our/the/?  I think the style guide has been steering away from this kind of personification...